hspec-core/src/Test/Hspec/Core/Clock.hs

```diff
@@ -28,8 +28,8 @@ toMicroseconds (Seconds s) = floor (s * 1000000)
 
 getMonotonicTime :: IO Seconds
 getMonotonicTime = do
```

Comment:
Maybe this shouldn't be called `getMonotonicTime` anymore, since this is not monotonic anymore. Maybe `getTime`?

Reply:
done